src/components/PlacementResult.js

```diff
@@ -12,13 +12,16 @@ type PlacementResultProps = {
   imageUrl: string | null;
   /** The raw html markup to display. Optional. */
   markup: string | null;
+  /** The position of the markup element in the placements array */
```

Comment:
I think we can get rid of the markupCount property... It's really only needed when creating the key to use for a possible array of `PlacementResult` components. Including it in the `<iframe>`'s title doesn't really mean anything to the user...